code/datums/outfits/ert/ap_eridani.dm

```diff
@@ -0,0 +1,162 @@
+		/obj/item/handcuffs/ziptie = 1,
+		/obj/item/melee/baton/stunrod = 1,
+		/obj/item/device/flashlight/maglight = 1,
+		/obj/item/shield/riot/tact = 1,
+		/obj/item/grenade/flashbang = 1
+	)
+
+/datum/outfit/admin/ert/ap_eridani/get_id_access()
+	return get_distress_access()
+
+/datum/outfit/admin/ert/ap_eridani/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if(visualsOnly)
+		return
+
+	if(H?.w_uniform)
+		var/obj/item/clothing/uniform = H.w_uniform
+		var/obj/item/clothing/accessory/sleevepatch/erisec/armband = new(src)
+		uniform.attach_accessory(null, armband)
+
+	if(!H.shoes)
+		var/obj/item/clothing/shoes/jackboots/toeless/shoes = new(src)
+		H.equip_to_slot_if_possible(shoes, slot_shoes)	
+
+/datum/outfit/admin/ert/ap_eridani/commander
+	name = "Eridani Section Commander"
+
+	uniform = /obj/item/clothing/under/rank/security/eridani/alt
+	suit = null
+	head = /obj/item/clothing/head/beret/security/eri
+	mask = /obj/item/clothing/mask/smokable/cigarette/cigar/cohiba
+	back = /obj/item/rig/strike/distress
+	suit_store = null
+	l_hand = /obj/item/gun/energy/rifle
+	r_hand = /obj/item/flame/lighter/zippo
+	l_pocket = /obj/item/clothing/mask/gas/tactical
+	r_pocket = /obj/item/storage/box/fancy/cigarettes/cigar
+
+	backpack_contents = list()
+
+	belt_contents = list(
+		/obj/item/ammo_magazine/c45x = 3,
+		/obj/item/device/flash = 1,
+		/obj/item/handcuffs/ziptie = 1,
+		/obj/item/melee/baton/stunrod = 1,
+		/obj/item/grenade/flashbang = 1,
+		/obj/item/shield/riot/tact = 1,
+		/obj/item/melee/telebaton = 1
+	)
+
+/datum/outfit/admin/ert/ap_eridani/doctor
+	name = "Eridani Medical Officer"
+
+	uniform = /obj/item/clothing/under/rank/eridani_medic
+	suit = /obj/item/clothing/suit/storage/medical_chest_rig
+	suit_store = /obj/item/clothing/head/hardhat/first_responder
+	head = /obj/item/clothing/head/beret/security/eri
+	mask = /obj/item/clothing/mask/surgical
+	glasses = /obj/item/clothing/glasses/hud/health/aviator
+	shoes = /obj/item/clothing/shoes/swat/ert
+	gloves = /obj/item/clothing/gloves/latex/nitrile
+	belt = /obj/item/storage/belt/medical
+	back = /obj/item/storage/backpack/satchel_med
+	accessory = /obj/item/clothing/accessory/holster/thigh
+	accessory_contents = list(/obj/item/gun/energy/disruptorpistol/magnum = 1)
+	r_ear = /obj/item/device/flashlight/pen
+	l_pocket = /obj/item/device/healthanalyzer
+	r_pocket = /obj/item/melee/telebaton 
+	l_hand = /obj/item/clothing/head/helmet/space/void/medical
+	r_hand = /obj/item/clothing/suit/space/void/medical
+
+
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
+		/obj/item/storage/firstaid/adv = 1,
+		/obj/item/storage/firstaid/surgery = 1,
+		/obj/item/device/advanced_healthanalyzer = 1,
+		/obj/item/clothing/accessory/storage/pouches/black = 1,
+		/obj/item/reagent_containers/glass/bottle/thetamycin = 1,
+		/obj/item/surgery/scalpel/manager = 1
+	)
+
+	belt_contents = list(
+		/obj/item/reagent_containers/hypospray/combat/empty,
```

Comment:
```suggestion
		/obj/item/reagent_containers/hypospray/combat/empty = 1,
```

This runtimes on spawn otherwise, and doesn't spawn any items in the belt.

Reply:
o missed that